src/ensembl/src/shared/components/instant-download/instant-download-transcript/InstantDownloadTranscript.tsx

```diff
@@ -110,6 +113,20 @@ const InstantDownloadTranscript = (props: Props) => {
     setTranscriptOptions(filterTranscriptOptions(so_term));
   }, [so_term]);
 
+  const onSubmit = async () => {
```

Comment:
Same as in InstantDownloadProtein — there is no point in making a function async if it isn't awaiting for anything. And there is no point in awaiting if no subsequent logic is depending on the awaited result 🤷‍♂️

Reply:
✅